### ISSUE TITLE
Suggestion: use application/json as Content-Type when format=json

### ISSUE THREAD

**Issue by chad-iris** [EIDA/routing#53]
With version 1.1.1, deployed at EIDA nodes, the `Content-Type` header returned for a request with `format=json` is set to:
```
Content-Type: text/plain; charset=UTF-8
```

As a convenience for web clients, you may consider setting this to:
```
Content-Type: application/json
```

**Comment by chad-iris** [comment]
```
$ curl -o out -v 'http://www.orfeus-eu.org/eidaws/routing/1/query?network=GE&service=dataselect&format=json'
...
> GET /eidaws/routing/1/query?network=GE&service=dataselect&format=json HTTP/1.1
> Host: www.orfeus-eu.org
> User-Agent: curl/7.54.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Wed, 12 Sep 2018 15:46:23 GMT
< Server: Apache/2.4.6 (Red Hat Enterprise Linux) PHP/5.4.16 mod_wsgi/3.4 Python/2.7.5
< Content-Length: 210
< Content-Type: text/plain; charset=UTF-8
< Access-Control-Allow-Origin: *
< 
...
```

**Comment by chad-iris** [comment]
Hrrm, I see this is known and documented:
https://github.com/EIDA/routing/blob/master/doc/specification.rst#output-json-format

There is no explanation of why though.  If there is a reason for not using the `application/json` it is worth considering adding it to the docs.

**Comment by javiquinte** [comment]
Fixed in f7a19f71b750c0ee80efb6434cd3f3c7ad4d4211
